docs/src/main/sphinx/release/release-458.md

```diff
@@ -0,0 +1,108 @@
+
+* {{breaking}} Deactivate [legacy file system support](file-system-legacy) for
+  all catalogs. You must activate the desired [file system
+  support](file-system-configuration) with
+  `fs.native-azure.enabled`,`fs.native-gcs.enabled`, `fs.native-s3.enabled`, or
+  `fs.hadoop.enabled` in each catalog. Use the migration guides for [Azure
+  Storage](fs-legacy-azure-migration), [Google Cloud
+  Storage](fs-legacy-gcs-migration), and [S3](fs-legacy-s3-migration) to assist
+  if you have not switched from legacy support. ({issue}`23343`)
+* Add JMX monitoring to the native S3 file system support. ({issue}`23177`)
+* Reduce the number of file system operations when reading tables with file system
+  caching enabled. ({issue}`23327`)
+* Improve the `flush_metadata_cache` procedure to include flushing the file
+  status cache. ({issue}`22412`)
+* Fix listing failure when Glue contains Hive unsupported tables. ({issue}`23253`)
+
+## Hudi connector
+
+* {{breaking}} Deactivate [legacy file system support](file-system-legacy) for
+  all catalogs. You must activate the desired [file system
+  support](file-system-configuration) with
+  `fs.native-azure.enabled`,`fs.native-gcs.enabled`, `fs.native-s3.enabled`, or
+  `fs.hadoop.enabled` in each catalog. Use the migration guides for [Azure
+  Storage](fs-legacy-azure-migration), [Google Cloud
+  Storage](fs-legacy-gcs-migration), and [S3](fs-legacy-s3-migration) to assist
+  if you have not switched from legacy support. ({issue}`23343`)
+* Add JMX monitoring to the native S3 file system support. ({issue}`23177`)
+
+## Iceberg connector
+
+* {{breaking}} Deactivate [legacy file system support](file-system-legacy) for
+  all catalogs. You must activate the desired [file system
+  support](file-system-configuration) with
+  `fs.native-azure.enabled`,`fs.native-gcs.enabled`, `fs.native-s3.enabled`, or
+  `fs.hadoop.enabled` in each catalog. Use the migration guides for [Azure
+  Storage](fs-legacy-azure-migration), [Google Cloud
+  Storage](fs-legacy-gcs-migration), and [S3](fs-legacy-s3-migration) to assist
+  if you have not switched from legacy support. ({issue}`23343`)
+* Add JMX monitoring to the native S3 file system support. ({issue}`23177`)
+
+## MariaDB connector
+
+* Reduce data transfer from remote systems for queries with large `IN` lists. ({issue}`23381`)
+
+## MySQL connector
+
+* Reduce data transfer from remote systems for queries with large `IN` lists. ({issue}`23381`)
+
+## Oracle connector
+
+* Reduce data transfer from remote systems for queries with large `IN` lists. ({issue}`23381`)
+
+## PostgreSQL connector
+
+* Reduce data transfer from remote systems for queries with large `IN` lists. ({issue}`23381`)
+
+## Redshift connector
+
+* Reduce data transfer from remote systems for queries with large `IN` lists. ({issue}`23381`)
+
+## SingleStore connector
+
+* Reduce data transfer from remote systems for queries with large `IN` lists. ({issue}`23381`)
+
+## Snowflake connector
```

Comment:
@raunaqmorarka this seems wrong .. a least in the docs there is nothinfg about domain compaction.

Reply:
SF connector is extending JdbcPlugin, so it gets this functionality by default

Reply:
https://github.com/trinodb/trino/pull/23463